### PR TITLE
Update WG_NAME_sandbox_stage.md

### PR DESCRIPTION
**Pull request by SecurityCRob**
suggested updates to wg sandbox template

### CODE REVIEW

**Review by marcelamelara — APPROVED** [review]

File: process/templates/WG_NAME_sandbox_stage.md [hunk 1]
```diff
@@ -14,9 +14,13 @@ When contributing to OpenSSF any existing material for the new WG to work on, th
   
 ### Working Group References
```
Comment (marcelamelara) [inline process/templates/WG_NAME_sandbox_stage.md:15]: Actually I missed something. Per #262, I thought we had made having a TAC sponsor a requirement for WGs at the sandbox stage: https://github.com/ossf/tac/blob/main/process/TI-Gives%2BGets.md#givesrequirements

**Review by marcelamelara — COMMENTED** [review]
I think we still need some re-syncing on the TAC sponsor requirements.

**Comment by camaleon2016** [comment]
We should consider at least a TAC "advisor" at the stage. maybe a sponsor incubating but definitely an "advisor" of sorts.

File: process/templates/WG_NAME_sandbox_stage.md [hunk 2]
```diff
@@ -14,9 +14,13 @@ When contributing to OpenSSF any existing material for the new WG to work on, th
```
Comment (steiza) [inline process/templates/WG_NAME_sandbox_stage.md:14]:
```suggestion
  
### TAC Sponsor
TAC sponsor agrees to attend WG meetings regularly, although they are not required to have a formal role in WG.
  * "name of TAC sponsor"
  
```

To address @marcelamelara's comment below

File: process/templates/WG_NAME_sandbox_stage.md [hunk 3]
```diff
@@ -14,9 +14,13 @@ When contributing to OpenSSF any existing material for the new WG to work on, th
...
+| Reference           | URL |
+|---------------------|-----|
+| Repo                |     |
+| Meeting Agenda      |     |
+| OSSF Calendar Entry |     |
+| Website             |     |
+| Contributing guide  |     |
+| Security.md         |     |
+| Roadmap             |     |
+| Other               |     |
```
Comment (steiza) [inline process/templates/WG_NAME_sandbox_stage.md:26]:
```suggestion

| Reference           | URL |
|---------------------|-----|
| Repo                |     |
| Website             |     |
| Security.md    |     |
| Other               |     |
```

According to https://github.com/ossf/tac/blob/main/process/working-group-lifecycle.md, once a group has become Sandbox, *then* it sets up regular meetings and gets on the OpenSSF calendar.

I initially didn't think a `Security.md` was part of that, but it looks like it is listed on https://github.com/ossf/tac/blob/main/process/TI-Gives%2BGets.md#givesrequirements.

**Review by sevansdell — APPROVED** [review]

**Review by sevansdell — APPROVED** [review]

File: process/templates/WG_NAME_sandbox_stage.md (hunk 1 above)
Comment (sevansdell) [reply process/templates/WG_NAME_sandbox_stage.md:19]: Agreed...this was helpful for the AIML WG.

**Review by steiza — APPROVED** [review]
This looks good-to-go to me!

**Review by mlieberman85 — APPROVED** [review]
LGTM

**Comment by SecurityCRob** [comment]
5 out of 9 TAC members have approved this, merging.

**Pull request merged**